### ISSUE TITLE
Review Codepen for Federal Header

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#16571]
## Description

Implementation of the Federal Header has been completed to the point where a review should be done, as was done in the previous [Footer ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16201). Notes:

- I'm using DS classes as I did in the footer again. I'm actually using these much more often and less CSS than I did in the Footer file.
- You can ignore internal/external link behavior again. In the header, everything is opening in the same tab but again, I am waiting additional guidance on this so I will update these at a later point.
- There are some icons missing, or have incorrect colors, etc. which I am aware of. I don't have access to these at the moment so we can update these later.
- This is responsive, you are able to review desktop and mobile.
- This is going to require a lot of copy/pasting of menus (unless FE engineers have a better way?) so for now not everything works, because I didn't want us to have to manually change everything if you found something you wanted to edit. The functional items, on both desktop & mobile, are:
  - Contact us link
  - Search button - but it only opens the flyout. The search field itself does not function
  - VA Benefits & Health Care tier 1 Button
    - Health Care tier 2 button and all links inside this menu
    - Disability tier 2 button and all links inside this menu
  - About VA tier 1 button ONLY
  - Find a VA location link
- Because we have to finish building the mega nav and there are probably going to be edits from phase 1 research it's going to be easier if have a header file that is not imported yet. For that reason I have 2 headers:
  - [The non-imported file](https://cdpn.io/pen/debug/PoVggmp/cd965bc015c83de0302f0f72dbc960d0) you probably want to look at to review the code itself
    - If you do make any edits, you can make them in here
  - [A full proto type screen](https://cdpn.io/pen/debug/QWowZvE/16ccaedf5ded20f8161208b973dde5d1) if you wanted to see how this will look on a fully completed page within the prototype but it pulls in a [separate, duplicate header file](https://codepen.io/Sitewide-Team/pen/abModNV/ac9526217476cb0517cfdf3b2e8a1879)
    - To be clear, this is not how the complete prototype will be set up, we won't have duplicate files for the same content, I'm just doing this temporarily

### Issues
These are things that we should probably solve, I'm not sure if it would require redoing any of the build that I currently have here or just some JS/CSS work:
- VA Benefits and Health Care menu - the 2 pieces are not the same height, the first white section is taller than the gray. I'm not necessarily sure if this is an issue or not, I'm just noting it as a difference from the current site. Probably should run this by Amanda.
- Multiple menus can be open at the same time, we should have this set up so that the old menus closes when the user opens a new menu.

**Reminder**
You need to be logged into the Sitewide CodePen account in order to edit the files. This is under Jill's login.

## Acceptance Criteria
- [x] Front end engineer has reviewed and updated the header as needed so that it is in a state that makes it as easy as possible to implement on VA.gov
- [x] Header is completed
- [x] UX Lead (Amanda) or PO review is requested 
- [x] Requires accessibility review

## Team
Please check the team(s) that will do this work.

- [ ] CMS Team
- [x] Public Websites
- [ ] Facilities
- [ ] Accelerated Publishing

**Comment by FranECross** [comment]
@chriskim2311 Although this hasn't been refined yet, I moved it to Stretch/Next Sprint and we can refine async (realizing Randi is OOO until after New Years).  cc @jilladams @randimays

**Comment by aklausmeier** [comment]
@laflannery Will this review ticket also include revisions based on research insights? I need to know which ticket to add updates needed. Minimal but want to make sure I add them to the right ticket.

**Comment by cindymerrill** [comment]
@aklausmeier What design changes have been made since the Figma prototypes?

**Comment by laflannery** [comment]
@aklausmeier You can add updates to this ticket since everything will need to be done this sprint anyway I think that should be fine

**Comment by laflannery** [comment]
@cindymerrill As noted in the ticket description, the header is not complete. Can you please move your comment to the screens tickets, which is the appropriate ticket for that work?

**Comment by cindymerrill** [comment]
Happy to move my comment, but which ticket is that, @laflannery ?

Also wondering if you could please reply to my question in [slack](https://dsva.slack.com/archives/C52CL1PKQ/p1704216901124899?thread_ts=1703716392.350969&cid=C52CL1PKQ): 

> That makes me wonder whether the CodePen will be able to indicate to a screen reader that something is a link without actually going to a new page. This might be a question for [@Laura Flannery](https://dsva.slack.com/team/U03JU991LCW); I’d like for research participants to tell me what they want to click on even if the link doesn’t go to a new page. Is that possible to do--make something “look” clickable to a screen reader but not actually go anywhere?

**Comment by aklausmeier** [comment]
@laflannery Header Revisions

1. USA banner is now a full color logo on white background, 1px #000000 line top and bottom
![Screenshot 2024-01-04 at 6 47 55 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/137226108/7d4898cd-a495-4ec1-9183-39338e1ac556)

2. New USA logo file that includes "Government" as part of the logo, header uses color version, emailed files to Laura
3. All fonts should be using Public Sans

cc @cindymerrill FYA - USA logo at top is visually changing, no impact to research goal of collecting sentiment but we will want to note this change in your synthesis of Phase 2 compared to Phase 1

**Comment by cindymerrill** [comment]
@aklausmeier What will the new USA logo look like in the footer? Black on a white background?

**Comment by laflannery** [comment]
Per amanda:
US Department of Veteran Affairs in header logo should be Public Sans text and not an image.
I will get you the new seal image today as planned.

**Comment by laflannery** [comment]
@aklausmeier Revisions 1 & 2 are made

**Comment by randimays** [comment]
Status update: I went in and fixed the JS issues Laura mentioned, and did a bit more cleanup. I did not see any glaring issues otherwise. There are some opportunities to clean up the mega menu styles and finesse the opening and closing of menus within it, but since that is not in our scope for now, I think we can save it for later.

**Comment by laflannery** [comment]
Randi and Chris have reviewed so I'm reassigning to myself to finish this up

**Comment by laflannery** [comment]
Dave C reviewed and signed off on full prototype including the header during our 1/12 meeting
